docs/tools/kubernetes.md

```diff
@@ -2,114 +2,39 @@
 title: Aiven Operator for Kubernetes®
 ---
 
-Manage Aiven services with [Aiven Operator for Kubernetes®](https://aiven.github.io/aiven-operator/) by using [Custom Resource Definitions (CRD)](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/).
+Manage Aiven infrastructure with [Aiven Operator for Kubernetes®](https://github.com/aiven/aiven-operator/) by using [Custom Resource Definitions (CRD)](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/).
 
-Aiven Operator for Kubernetes® supports:
-
-- Aiven for Apache Kafka®
-- Aiven for Caching
-- Aiven for ClickHouse®
-- Aiven for OpenSearch®
-- Aiven for PostgreSQL®
+The [Aiven Operator documentation](https://aiven.github.io/aiven-operator/index.html) includes an API reference and example usage for the resources.
```

Comment:
Nitpicking: Consider starting this piece of info with its actual value added for the user, e.g. `For example usage of the resources and API referenece, see the Aiven Operator documentation.`.